workflow/controller/agent.go

```diff
@@ -128,6 +164,9 @@ func (woc *wfOperationCtx) createAgentPod(ctx context.Context) (*apiv1.Pod, erro
 					Image:           woc.controller.executorImage(),
 					ImagePullPolicy: woc.controller.executorImagePullPolicy(),
 					Env:             envVars,
+
+					VolumeMounts: woc.getCertVolumeMount(ctx, "argo-workflows-agent-ca-certificates"),
```

Comment:
move cont to `common.CACertificatesVolumeMountName`

Reply:
done